Packs/Pcysys/Playbooks/playbook-Pentera_Filter_And_Create_Incident.yml

```diff
@@ -0,0 +1,1250 @@
+              right:
+                value:
+                  simple: Ability to access an open share
+              ignorecase: true
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 1962.5,
+          "y": 1330
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "25":
+    id: "25"
+    taskid: 7d3b5789-9dce-49a5-8232-47040ef4e08b
+    type: condition
+    task:
+      id: 7d3b5789-9dce-49a5-8232-47040ef4e08b
+      version: -1
+      name: Are we filtering open shares
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "8"
+      "yes":
+      - "24"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: 'inputs.Filter open shares '
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+    view: |-
+      {
+        "position": {
+          "x": 1850,
+          "y": 1155
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "26":
+    id: "26"
+    taskid: 2466a731-6e00-4f30-8a8b-6079bd4d92ed
+    type: regular
+    task:
+      id: 2466a731-6e00-4f30-8a8b-6079bd4d92ed
+      version: -1
+      name: Pentera Group Incidents
+      scriptName: PenteraOperationToIncident
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "18"
+    scriptarguments:
+      context_key: {}
+      custom_fields_output:
+        simple: penteraoperationdetails
+      full_action_report:
+        complex:
+          root: FilterPenteraOperations
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 1050,
+          "y": 1860
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 2295,
+        "width": 2592.5,
+        "x": -250,
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: Filter password cracking
+  value:
+    simple: "True"
+  required: true
+  description: Filter all password cracking operation types. Any other value will be considered as "False"
+  playbookInputQuery: null
+- key: Filter BlueKeep vulnerability
+  value:
+    simple: "True"
+  required: true
+  description: Filter all BlueKeep operation types. Any other value will be considered as "False"
+  playbookInputQuery: null
+- key: Filter Minimum Severity
+  value:
+    simple: "0"
+  required: true
+  description: "Filters actions according to given value. \nWill not filter anything
+    if got zero (0) as value."
+  playbookInputQuery: null
+- key: FullActionReport
```

Comment:
If this input is provided by the search playbook than they should be wrapped in a parent playbook together or this value wont get populated. Or maybe it would be better to leave it empty and have the user input it. Anyway the description should be improved a bit.